<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>WriteMailBase</class>
 <widget class="QWidget" name="WriteMailBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form1</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>2</number>
   </property>
   <property name="spacing" >
    <number>2</number>
   </property>
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>2</number>
     </property>
     <item>
      <widget class="QLabel" name="toLabel" >
       <property name="text" >
        <string>To</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="ccLabel" >
       <property name="text" >
        <string>CC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="subjectLabel" >
       <property name="text" >
        <string>Subject</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fromLabel" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string>From</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bccLabel" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="text" >
        <string>BCC</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>2</number>
     </property>
     <item row="0" column="1" >
      <widget class="QToolButton" name="toPicker" >
       <property name="toolTip" >
        <string/>
       </property>
       <property name="whatsThis" >
        <string>Select recipients from the addressbook.</string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLineEdit" name="subjectInput" />
     </item>
     <item row="3" column="0" >
      <widget class="QComboBox" name="fromBox" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="whatsThis" >
        <string>Select the from address used for this mail.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLineEdit" name="ccInput" />
     </item>
     <item row="4" column="1" >
      <widget class="QToolButton" name="bccPicker" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="toolTip" >
        <string/>
       </property>
       <property name="whatsThis" >
        <string>Select blind carbon copies from the addressbook.</string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QToolButton" name="ccPicker" >
       <property name="toolTip" >
        <string/>
       </property>
       <property name="whatsThis" >
        <string>Select carbon copies from the addressbook.</string>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" >
      <widget class="QLineEdit" name="bccInput" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLineEdit" name="toInput" />
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="AddAtt" name="addAtt" />
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QTextEdit" name="emailInput" >
     <property name="whatsThis" >
      <string>Enter the body text here.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>AddAtt</class>
   <extends></extends>
   <header>addatt.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
